get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
+|The Movie Graph |A mini graph model of connections between actors and movies |`:play movie graph`
+|The Northwind Database |A classic use case of RDBMS to graph with import instructions and queries |`:play northwind graph`
+|Custom Guides |Starting with Neo4j 2.3, you can use _:play <url>_ to play a custom guide |`:play
+
+http://guides.neo4j.com/graphgist/restaurant_recommendation.html`
+|===
+
+All you have to do is enter any of the above commands in the browser command line (at the top of the browser) and follow the accompanying guides to walk through the training or scenarios.
+To execute any of the queries provided within a browser guide, simply click on the query, and it will populate in the browser command line.
+Then, just press kbd:[Enter] or the Play button to execute the statement!
+Results of the query will populate in a panel right below the command line.
 
-image:://s3.amazonaws.com/dev.assets.neo4j.com/wp-content/uploads/neo4j-browser-sync.png[width=500]
+== Styling Neo4j Browser Visualization
+
+As mentioned above, any queries you run in Neo4j Browser will populate your results below the command line.
+Query results are rendered either as a visual graph, a table format, or an ascii-table result.
+You can switch between those with the icons on the left side of the result frame.
+If you cannot see the whole graph or the results display too close together, you can adjust by moving the visual view and dragging nodes to rearrange them.
+To move the view to see more parts of the graph, just click an empty spot within the graph pane and drag it. To rearrange nodes, you can click and drag them around.
+
+The nodes will already have sensible captions assigned by the browser, which auto-selects a property from the property list to use as a caption.
+You can see all the properties of that element if you click on any node or relationship.
+Properties will appear below the visualization.
+Larger property sets might be collapsed into a subset, but there is a little triangle on the right to unfold them.
+
+For example, if you click on one of the `Movie` nodes in the MovieGraph (`:play movie graph`), then you can see its properties below the graph visual.
+The same applies for `Actor` nodes or the `ACTED_IN` relationships.
+If you click on any label or relationship *above* the graph visualization, you can then choose its styling in the area below the graph.
+Colors, sizes, and captions are selectable from there.
+
+To see this for yourself, you can click on the `Person` label above the graph and change the color, size, and captions of all nodes labeled with `Person`.
+The first image below shows changes to nodes labeled `Person`. The second image shows changes to relationships labeled `ACTED_IN`.
+
+image::{img}style_person_node.png[]
 
-Just log in with your GitHub, Twitter or Google account and synchronize your workplace to the cloud.
+image::{img}style_actedin_relationship.png[]
 
```

Comment:
perhaps mention full screen and that you can zoom there and that it keeps the zoom level even if you go back?

Reply:
mention the "Auto connect nodes" in the settings for graph-viz
i.e. that it auto-fetches relationships between all nodes and if you don't want that, e.g. b/c you return select relationships you can disable it there.

Reply:
Full screen for the Browser window or full screen for the graph pane? Just want to make sure I put it in the right place. :)
Is that the "Connect result nodes" checkbox at the bottom? What are the cases where you would want to auto-fetch relationships versus not? Not sure I understand this functionality...

Reply:
full-screen graph pane.

usually you want to auto-fetch when you just explore and don't bother to return exact relationships or full paths

but if you need to examine production queries from an application you exactly want to see only what the query returned, then you can disable it.